universal-application-tool-0.0.1/app/models/Program.java

```diff
@@ -36,6 +36,9 @@
 
   @Constraints.Required private String description;
 
+  /** Link to external site for this program. */
```

Comment:
question (blocking): I'm so nervous for database updates - do we know what will happen to programs that do not currently have external links?

Reply:
The evolutions update should cover this. Any programs that don't currently have external links will default to the empty string.

`alter table programs add column external_link varchar default '';`

Then in `ProgramIndexView` if the external link is empty then we fall back to the application info page.

Reply:
Thanks Aaron!